qid-batch-runner.yml

```diff
@@ -41,7 +41,7 @@ spec:
           stdin: true
           env:
             - name: RABBITMQ_SERVICE_HOST
-              value: rabbitmq
+              value: rm-rabbitmq-client
```

Comment:
I think this is wrong. It's `rm-rabbitmq` everywhere else

Reply:
Nick that is correct - it has now been updated.